packages/calcite-components/.stylelintrc.cjs

```diff
@@ -1,13 +1,7 @@
 // @ts-check
 
 // ⚠️ AUTO-GENERATED CODE - DO NOT EDIT
-const customFunctions = [
-  "get-trailing-text-input-padding",
-  "medium-modular-scale",
-  "modular-scale",
-  "scale-duration",
-  "small-modular-scale"
-];
+const customFunctions = [];
```

Comment:
@jcfranco any clue why this keeps being removed? Its something happening in pre commit formatting or linting I think.